public/components/visualizations/charts/lines/line.tsx

```diff
@@ -239,5 +301,10 @@ export const Line = ({ visualizations, layout, config }: any) => {
     [config, layoutConfig.config]
   );
 
+  useEffect(() => {
```

Comment:
We've to remove this useEffect as it only has console comments

Reply:
Will do